### PR TITLE
some wallet tests

### PR DESCRIPTION
**Pull request by jolesbi**
wallet component tests

### CODE REVIEW

**Comment by jolesbi** [comment]
@faboweb this builds locally without error — maybe when you sort out the package.json issues this will be fine?

File: app/src/renderer/vuex/modules/index.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
-const files = require.context('.', false, /\.js$/)
-const funcs = {}
-files.keys().forEach((key) => {
-  if (key === './index.js') return
-  funcs[key.replace(/(\.\/|\.js)/g, '')] = files(key).default
+export default (opts) => ({
```
Comment (faboweb) [inline app/src/renderer/vuex/modules/index.js]: require.context is not available in jest (comment on that from the lead dev of jest: https://github.com/facebookincubator/create-react-app/issues/517#issuecomment-243488178)

**Comment by jolesbi** [comment]
ok i'll improve them!

**Comment by jolesbi** [comment]
too many issues with this branch. will remake the PR.

**Pull request closed without merging**